crates/bvs-vault-router/src/contract.rs

```diff
@@ -60,33 +60,69 @@ pub fn execute(
     }
 }
 
+#[cfg_attr(not(feature = "library"), entry_point)]
+pub fn migrate(deps: DepsMut, _env: Env, msg: MigrateMsg) -> Result<Response, ContractError> {
+    cw2::ensure_from_older_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
+    match msg {
+        MigrateMsg::MapVaults {} => migration::map_vaults(deps),
+    }
+}
+
+mod migration {
+    use crate::state::{OPERATOR_VAULTS, VAULTS};
+
+    use super::*;
+
+    pub fn map_vaults(deps: DepsMut) -> Result<Response, ContractError> {
+        let vaults = VAULTS
+            .range(deps.storage, None, None, cosmwasm_std::Order::Ascending)
```

Comment:
```suggestion
            .keys(deps.storage, None, None, cosmwasm_std::Order::Ascending)
```

I believe there should be a key only operation for this sort of things.